### DISCUSSION TITLE
Build Error For Android

### DISCUSSION THREAD

**Discussion by gorkemalyagut** [flutter-stripe/flutter_stripe#1677] [Q&A]
When I run the flutter build apk command, the error I get is as follows. I did all the necessary steps in the Stripe doc correctly, I don't understand why the error. 

When I go to the file extension in the error, the colors specified there are present, but how can it not find it?

**Smartphone / tablet**
 - Device: Pixel 7 Pro
 - OS: Android 14
 - Package version: '10.1.0
 - Flutter version 3.9.2

![Screen Shot 2024-03-04 at 14 43 28 PM](https://github.com/flutter-stripe/flutter_stripe/assets/57588583/2b4dff49-ccfb-4f4e-a2fe-34c76d7ef9af)

**Comment by devilaark2018** [comment]
same error form me also

**Comment by hardik-thirstydevs** [comment]
in my both Windows Computers, working fine in Android Studio but giving the above error if I run in VSCode.
Code is the same but not sure what is Android Studio is using but VSCode is not. 😕

Flutter 3.16.4
Android Studio: Android Studio Hedgehog | 2023.1.1 Patch 2

(Not tested in Other OS)

**Comment by gorkemalyagut** [comment]
I solved the problem using this and it works, but in the latest version of the package I get the error.

dependency_overrides:
  stripe_platform_interface: 10.0.0
  stripe_android: 10.0.0
  stripe_ios: 10.0.0
